doc/tutorials/introduction/crosscompilation/crosscompile_with_multiarch.markdown

```diff
@@ -0,0 +1,341 @@
+====================================
+
+@prev_tutorial{tutorial_arm_crosscompile_with_cmake}
+@next_tutorial{tutorial_building_tegra_cuda}
+
+[TOC]
+
+|    |    |
+| -: | :- |
+| Original author | Kumataro |
+| Compatibility   | Ubuntu >=23.04 |
+|^                | OpenCV >=4.8.0 |
+
+@warning
+This tutorial can contain obsolete information.
+
+What is "MultiArch"
+-------------------
+Ubuntu and Debian supports MultiArch.
+It allows to install several foreign architecture libraries.
+
+(e.g. aarch64 and/or armv7 binaries are able to installed at x86-64 hosts.)
+
+MultiArch can be used to cross-compile OpenCV library too.
+
+@warning
+- Following these steps will make your Linux environment a little dirty.
+  If possible, it is better to use VMs.
+- This tutorial expects host and target uses same ubuntu version.
+   Do not use/mix different versions for external library dependency.
+  - OK: Host and Target are 23.04.
+  - OK: Host and Target are 23.10.
+  - NG: Host is 23.04, and Target is 23.10.
+  - NG: Host is 23.10, and Target is 23.04.
+
+Downloading tools
+-----------------
+Install nessesary tools and toolchains to cross-compile.
+
+- git, cmake, build-essential are required basically.
+- ccache is to reduce compilation time(option).
+- ninja-build is to reduce compilation time(option).
+- libfreetype-dev and libharfbuzz-dev is for example to use external libraries(option).
+- crossbuild-essential-armhf is toolchain package for armv7 target.
+- crossbuild-essential-arm64 is toolchain package for aarch64 target.
+
+```
+sudo apt update -y
+sudo apt install -y \
+    git \
+    cmake \
+    build-essential \
+    ccache \
+    ninja-build \
+    libfreetype-dev \
+    libharfbuzz-dev \
+    crossbuild-essential-armhf \
+    crossbuild-essential-arm64
+```
+
+Working folder structure
+------------------------
+In this sample, following working folder structure are used.
+
+```
+/home
+  + kmtr                    - please replace your account name.
+    + work
+      + opencv              - source, cloned from github
+      + opencv_contrib      - source, cloned from github
+      + build4-full         - artifact(for Host), created by cmake
+      + build4-full_arm64   - artifact(for aarch64 target), created by cmake
+      + build4-full_armhf   - artifact(for armhf target), created by cmake
+```
+
+1. Create working folder under your home directory.
+2. Clone OpenCV and OpenCV Contrib from repository to work directory.
+
+```
+cd ~
+mkdir work
+cd work
+git clone https://github.com/opencv/opencv.git
+git clone https://github.com/opencv/opencv_contrib.git
+```
+
+Self-compile(for x86-64)
+------------------------
+
+Following are to compile for host(x86-64) at host(x86-64).
+
+```
+[Host]
+cmake -S opencv \
+      -B build4-full \
+      -DOPENCV_EXTRA_MODULES_PATH=opencv_contrib/modules \
+      -GNinja
+
+     cmake --build   build4-full
+sudo cmake --install build4-full
+sudo ldconfig
+```
```

Comment:
x86 is not needed for cross-compilation

Reply:
I has removed host compile section, thank you !